build.sbt

```diff
@@ -42,13 +42,14 @@ lazy val root = project
     interopCatsJVM,
     interopCatsJS,
     interopFutureJVM,
-//  interopMonixJVM,
-//  interopMonixJS,
+    interopMonixJVM,
+    interopMonixJS,
     interopScalaz7xJVM,
     interopScalaz7xJS,
     interopJavaJVM,
     interopReactiveStreamsJVM,
-//  benchmarks,
+    interopTwitterJVM,
+    benchmarks,
```

Comment:
What was the reason to have `benchmarks` commented out?

Reply:
No idea. It was commented out for a long time now. As far as I can see it's commented out together with Monix during "death of TF" PR :).

Reply:
I remember it :D , I wondered why it was already commented before